core/server/services/members/middleware.js

```diff
@@ -69,6 +69,57 @@ const getOfferData = async function (req, res) {
     });
 };
 
+const getMemberNewsletters = async function (req, res) {
+    try {
+        const memberUuid = req.query.uuid;
```

Comment:
We should probably guard against this existing before passing it to the members service - I'm not sure what happens if we pass `{uuid: undefined}` to `members.get`

Reply:
✅